sympy/core/expr.py

```diff
@@ -2877,6 +2877,57 @@ def _eval_nseries(self, x, n, logx):
                      nseries calls it.""" % self.func)
                      )
 
+    def aseries(self, x, n=6, bound=0, hierar=False):
+        from sympy.series.gruntz import mrv, rewrite, mrv_leadterm
+        from sympy.functions import exp, log
+
+        if x.is_positive is x.is_negative is None:
+            xpos = C.Dummy('x', positive=True, bounded=True)
```

Comment:
C is not defined here. If this is supposed to be the class registry, that is deprecated (just import Dummy).